readme.txt

```diff
@@ -1,36 +1,70 @@
-=== Font Awesome ===
-Contributors: fontawesome, mlwilkerson, kemitchell
+=== Font Awesome Official ===
+Contributors: fontawesome, mlwilkerson, kemitchell, robmadole
+Stable tag: 0.2.0
 Tags: font, awesome, fontawesome, font-awesome, font-awesome-official, icon, svg, webfont
 Requires at least: 4.7
-Tested up to: 4.9.8
+Tested up to: 5.0
 Requires PHP: 5.6
 License: GPLv2 or later
 License URI: https://www.gnu.org/licenses/gpl-2.0.html
 
-Loads Font Awesome 5 Free or Pro, managing version resolution across multiple client themes or plugins.
+Adds Font Awesome 5 icons to your WordPress site. Supports Font Awesome Pro. Resolves conflicts across many plugins or themes that use Font Awesome.
 
 == Description ==
 
-Loads Font Awesome 5 Free or Pro on your WordPress site. Provides features for developers of themes and other plugins
-to register their Font Awesome requirements with this plugin to ensure that a single, working version of Font Awesome
-is loaded that works across all WordPress components. Optionally, block some themes or plugins from trying to load
-their own versions of Font Awesome which might create conflicts and keep your icons from working correctly.
+Adds Font Awesome 5 icons to your WordPress site. Supports Font Awesome Pro. Resolves conflicts across many plugins or themes that use Font Awesome.
 
-Supports both Free and Pro, optional use of version 4 compatibility (shims), pseudo-elements, and either the [Webfont
-with CSS](https://fontawesome.com/how-to-use/on-the-web/setup/getting-started?using=web-fonts-with-css) or
-[SVG with JavaScript](https://fontawesome.com/how-to-use/on-the-web/setup/getting-started?using=svg-with-js) method.
+See also the GitHub [README](https://github.com/FortAwesome/wordpress-fontawesome/README.md).
 
-Loads Font Awesome from the official Font Awesome Free or Pro CDN.
+## Features
 
-[Find more documentation here.](https://github.com/FortAwesome/wordpress-fontawesome/README.md)
+1. Supports both Font Awesome Free and Font Awesome Pro.
```

Comment:
I added a Features list. This is the content that would appear in the plugin directory entry.